src/modules/did/did.service.ts

```diff
@@ -90,13 +90,18 @@ export class DIDService {
    */
   public async getById(did: string): Promise<IDIDDocument> {
     const cachedDIDDocument = await this.didRepository.findOne(did);
-    if (cachedDIDDocument) return cachedDIDDocument;
+    if (cachedDIDDocument) {
+      delete cachedDIDDocument.logs;
```

Comment:
rather the returning the entire object and deleting properties we don't need, I think we could just select the fields that we need and return that.

Reply:
@ahmedolaibrahim good point. In that way, we can be sure that we are constructing the return object that we want